src/base/base.c

```diff
@@ -1384,6 +1371,15 @@ void xdebug_base_post_startup()
 
 void xdebug_base_rinit()
 {
+	XG_BASE(error_reporting_override)= 0;
+	XG_BASE(error_reporting_overridden) = 0;
+	XG_BASE(output_is_tty) = OUTPUT_NOT_CHECKED;
+
+	XG_BASE(private_tmp) = NULL;
```

Comment:
This variable doesn't seem to be written at the request-time, so a true global set in MINIT might be possible.